source/log-parser/log-parser.py

```diff
@@ -29,7 +29,7 @@
 # Constants
 #======================================================================================================================
 API_CALL_NUM_RETRIES = 3
-BLOCK_ERROR_CODES = ['400','403','404','405'] # error codes to parse logs for
+BLOCK_ERROR_CODES = environ['BLOCK_ERROR_CODES']# error codes to parse logs for
```

Comment:
what happens if this is not set?

Reply:
may be set a default value?

Reply:
@arunsivasankaran default value is set in the terraform template